internal/controller/constants.go

```diff
@@ -21,6 +21,7 @@ import "time"
 const (
 	OrgInstaslicePrefix              = "instaslice.redhat.com/"
 	ManagedLabel                     = OrgInstaslicePrefix + "managed"
+	LabelInstasliceMutated           = OrgInstaslicePrefix + "mutated"
```

Comment:
Just to make code more readable, could you rename this to something like
```suggestion
	PodLabelInstasliceMutated           = OrgInstaslicePrefix + "mutated"
```